src/libxrpl/protocol/RPCErr.cpp

```diff
@@ -28,7 +28,7 @@ struct RPCErr;
 
 // VFALCO NOTE Deprecated function
 Json::Value
-rpcError(int iError)
+rpcError(error_code_i iError)
```

Comment:
Is it even still used? says it's deprecated above so maybe possible to remove entirely. Worth checking